backend/tests/test_lifespan.py

```
@@ -22,94 +22,71 @@ def test_app():
         async with lifespan(test_app):
             mock_logger.error.assert_called_with(
-                f"Failed to start containers: {startup_error}"
+                f"Failed to start application: {startup_error}"
             )
 
 
-@pytest.mark.asyncio
-async def test_lifespan_shutdown_failure(mock_logger, test_app):
-    """Test error handling during container shutdown"""
-    shutdown_error = Exception("Container shutdown failed")
+from unittest.mock import Mock, patch
```

Ruff (0.8.2)

Redefinition of unused `Mock` from line 2

Remove definition: `Mock`

(F811)